src/gas-api.js

```diff
@@ -1,210 +1,212 @@
-"use strict";
+'use strict';
 
-function pull(code) {
-  const changed = $('.diff-file:checked').toArray().map(e => e.value);
-  const update_promises = changed.filter(f => code.github[f])
-  .map((file) => {
-    const match = file.match(/(.*?)\.(gs|html)$/);
-    if (!match || !match[1] || !match[2]) {
-      showAlert('Unknow Error', LEVEL_ERROR);
-      return;
-    }
-    const name = match[1];
-    const type = match[2];
+const Gas = class Gas {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/9b7bf2b597e67365f1e9002c784237b58657e4cc/src/gas-api.js#L3"></span>‘Gas’ is assigned a value but never used.</td>
    </tr>
  </tbody>
</table>